src/croniter/croniter.py

```diff
@@ -83,6 +84,20 @@ def is_32bit():
 for i in M_ALPHAS, DOW_ALPHAS:
     ALPHAS.update(i)
 del i
+
+ExpressionAlias = namedtuple('ExpressionAlias', 'regular, hashed')
+
+# Non-standard expression aliases
+EXPR_ALIASES = {
```

Comment:
Also pulls this up into the module scope so we're not redeclaring this mapping every time

Reply:
Damn, the above comment was not validated, so you didnt saw it... sorry :(

Reply:
> users shall be allowed to update easily the aliases.

What makes it difficult to update the aliases because we're using a named tuple? Could you share an example of what you're thinking that is more difficult?

Reply:
dict are mutable, not namedtuple.

Reply:
You could just replace it like this though right

```py
# Replace alias with whatever you want
EXPR_ALIASES['@midnight'] = ExpressionAlias("1 2 * * *", "h h * * *")
```